radio/src/targets/common/arm/stm32/rtc_driver.cpp

```diff
@@ -48,7 +48,7 @@ uint32_t getRTCBKPR(uint8_t RTCBKPRegister) {
     RTC->BKP19R = RTCCHKSUM();
   }
 
-  uint32_t value = ((uint32_t *)RTC)[RTCBKPRegister];
+  uint32_t value = ((uint32_t *)RTCBKBR_BASE)[RTCBKPRegister];   // RTC backup registers are 32bit registers
```

Comment:
It would be really better to use the functions defined in HAL/LL rather than our own for better compatibility with future MCUs. See my comments in #3837.

Reply:
done in 2.10 port